core/src/main/java/com/graphhopper/routing/ev/RouteNetwork.java

```diff
@@ -27,10 +27,8 @@
 
     MISSING("missing"), INTERNATIONAL("international"), NATIONAL("national"), REGIONAL("regional"),
     LOCAL("local"), OTHER("other");
-
-    public static String key(String prefix) {
-        return prefix + "_network";
-    }
```

Comment:
What does this change improve? We have similar handling e.g. with EncodingManager.

Also please note my recent change here: https://github.com/graphhopper/graphhopper/commit/71631dfb480ebac8c9035c34e0e08b8cadc45949

Reply:
Java allows switch-case statements for Strings only if they're constants. A value obtained from a static method doesn't work.

Reply:
I see. But for this single switch-case usage I wouldn't want to change it globally. You could just assign it like you did for the priority key?

Reply:
Do we get any benefits by indirecting the key generation via `RouteNetwork.key()` ?

```java
public class BikeNetwork {
    public static final String KEY = RouteNetwork.key("bike");
}
```

```java
public class BikeNetwork {
    public static final String KEY = "bike" + RouteNetwork.KEY_SUFFIX;
}
```

At runtime there's no difference between these two but the first one is not compile time constant which blocks the usage in the switch statement.